blazor/diagram/ports/interaction.md

```diff
@@ -70,6 +71,97 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 
 ![Drawing Port Connection in Blazor Diagram](../images/blazor-diagram-draw-port-connection.gif)
 
+## How to draw different types of connector from the port
+
+The type of connector segment drawn from the port can be specified using the [DrawingObject](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Diagram.SfDiagramComponent.html#Syncfusion_Blazor_Diagram_SfDiagramComponent_DrawingObject) property. This enables the drawing of various connector types from port, including:
```

Comment:
The type of connector segment drawn from the port can be specified using the [DrawingObject](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Diagram.SfDiagramComponent.html#Syncfusion_Blazor_Diagram_SfDiagramComponent_DrawingObject) property. This enables the drawing of various connector types from the port, including: